core/js/src/main/scala/scalaz/zio/RTS.scala

```diff
@@ -0,0 +1,22 @@
+package scalaz.zio
+
+import scalaz.zio.internal.impls.EnvJS
+
+trait RTS {
+  lazy val env =
+    EnvJS.newDefaultEnv {
+      case cause if cause.interrupted => IO.unit // do not log interruptions
+      case cause                      => IO.sync(println(cause.toString))
```

Comment:
I know the browser has an uncaught error handler that could be used, not sure about node.js, perhaps there's something more idiomatic here than `println`?

Reply:
Nah, we shouldn't throw. Maybe we leave it like this.